scripts/style-spec.js

```diff
@@ -1,3 +1,6 @@
 var spec = module.exports = require('../mapbox-gl-js/src/style-spec/reference/v8');
 
 // Make temporary modifications here when Native doesn't have all features that JS has.
+delete spec.layout_symbol['symbol-sort-key'];
```

Comment:
nit: maybe worth creating backporting issue for gl-native.

Reply:
https://github.com/mapbox/mapbox-gl-native/issues/14028